recipes/scons/all/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, tools
+from io import StringIO
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "scons"
+
+    def build(self):
+        scons_path = tools.which("scons")
+        assert scons_path.replace("\\", "/").startswith(self.deps_cpp_info["scons"].rootpath.replace("\\", "/"))
+
+        output = StringIO()
+        self.run("{} --version".format(scons_path), run_environment=True, output=output)
+        text = output.getvalue()
+        print(text)
+        assert self.requires["scons"].ref.version in text
```

Comment:
Although I think none of them are documented, I think it will be *safer* get the version from `self.deps_cpp_info["scons"].version`

Reply:
I have addressed your concerns. Please review again.

Reply:
Ok, https://github.com/conan-io/conan-center-index/pull/814#issuecomment-598255464 happens so I revert it back to the original.

Reply:
This should be addressed since Conan 1.24 https://github.com/conan-io/conan/releases/tag/1.24.0

We can merge this and apply the fix later, just to keep in mind